Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -256,8 +264,9 @@ static bool webKitMediaOpenCDMDecryptorDecrypt(WebKitMediaCommonEncryptionDecryp
         gst_buffer_unmap(subSamplesBuffer, &subSamplesMap);
     } else {
         // Decrypt cipher.
-        GST_TRACE_OBJECT(self, "decrypting");
-        if ((errorCode = priv->m_openCdm->Decrypt(map.data, static_cast<uint32_t>(map.size),
+        GST_TRACE_OBJECT(self, "decrypting (no subsamples)");
+        auto decryptor = std::make_unique<media::OpenCdm>(keyIDBufferMap.data, keyIDBufferMap.size);
```

Comment:
Condidering that these are the two only places where priv->m_openCdm is used, I would just remove that attribute.